libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/pdfformatprovider.md

```diff
@@ -5,7 +5,7 @@ description: Using PdfFormatProvider
 slug: radpdfprocessing-formats-and-conversion-pdf-pdfformatprovider
 tags: using,pdfformatprovider
 published: True
-position: 2
+position: 1
```

Comment:
This change is related to the deletion of libraries/radpdfprocessing/formats-and-conversion/pdf/pdf.md